### ISSUE TITLE
Add Sonar FM

### ISSUE THREAD

**Issue by akrasiko** [web-scrobbler/web-scrobbler#3457]
Hi,

I would like to thank you all for Web Scrobbler. In addition to that, it would be great if you could add to the extension this site: https://sonarfm.cl/

**Comment by hornbuckle** [comment]
Unfortunately, there is nothing to scrobble from this site because it appears to be all podcasts without individual artist and track information.

**Comment by akrasiko** [comment]
Hi!
It's an online radio, such as Radio Horizonte (https://horizonte.cl/) which
is also incorporated in Web Scrobbler

Thanks!

**Comment by hornbuckle** [comment]
Okay, I checked the site again today and see that it does have the one player on the main page, like Horizonte, that may sometimes have artist and track info.
<img width="962" alt="" src="https://user-images.githubusercontent.com/6808988/200939908-5ac97e2f-2c48-45d6-80a2-3c611df9dee0.png">
When I had checked previously, it was not showing this and just showed the program/podcast information. It appears to be the same player as Horizonte, so I can look into adding support for Sonar FM and update both to use the same connector.
I have a couple other updates to submit soon and will try to add this, as well.
